Markr/EditorViewController.swift

```diff
@@ -193,17 +199,25 @@ class EditorViewController: NSViewController {
   }
   
   func loadMarkdown() {
-    let down = Down(markdownString: editor.string)
-    
     if let font = editor.font {
-      setFont(font: font)
+        setFont(font: font)
     }
     
-    if let attrMarkdown = try? down.toAttributedString(),
-      let splitViewController = self.parent as? NSSplitViewController,
-      let markdownSplitView = splitViewController.splitViewItems.last {
-      let markdownVC = markdownSplitView.viewController as? MarkdownViewController
-        markdownVC?.setMarkdown(attrMarkdown)
+    let string = editor.string
+    
+    DispatchQueue.global(qos: .userInitiated).async {
+        if let splitViewController = self.parent as? NSSplitViewController,
+            let markdownSplitView = splitViewController.splitViewItems.last {
+            if let markdownVC = markdownSplitView.viewController as? MarkdownViewController {
+                markdownVC.markdown.textStorage?.beginEditing()
```

Comment:
This line throws an error: 

```
Main Thread Checker: UI API called on a background thread: -[NSTextView textStorage]
```

Can we move this line to the main thread, we can begin editing on the main thread after the markdown has been parsed by Down:

```swift
DispatchQueue.main.async {
  markdownVC.markdown.textStorage?.beginEditing()
  markdownVC.setMarkdown(attrMarkdown)
}
```